packages/map/src/story_utils.jsx

```diff
@@ -0,0 +1,4 @@
+export const MAP_CREDENTIALS = {
```

Comment:
open for suggestions on a better filename

Reply:
can we avoid having this file in general? run DefinePlugin or something

Reply:
Those credentials are needed in almost every story. Not sure if we should simply inline it.

Since we want to move storybook config to root package, we should keep it general - therefore using DefinePlugin would add this 'define' to every subpackage's storybook

Reply:
maybe keep simple js file in the root local.js with all secrets and add alias in webpack for it.
no need to dotenv